observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,202 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+You can deploy and configure the OpenTelemetry Collector to forward logs to a LokiStack instance. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
+
+This use of the Loki Exporter is a temporary Technology Preview feature, which is planned to be deprecated with the publication of an improved solution in which the Loki Exporter is replaced with the OTLP HTTP Exporter.
```

Comment:
```suggestion
This use of the Loki Exporter is a temporary Technology Preview feature that is planned to be replaced with the OTLP HTTP Exporter.
```
Will a Loki Exporter have any support at all once OTLP is out? If not, I would just say "replaced" clearly. Deprecation indicates some sort of continued minimal support prior to removal IMHO.